src/profile/mfa/mfa_factor_sources/security_questions_factor_source/kdf/is_security_questions_kdf_scheme.rs

```diff
@@ -0,0 +1,8 @@
+use crate::prelude::*;
+
+pub trait IsSecurityQuestionsKDFScheme {
```

Comment:
Couldn't trait just be named  `SecurityQuestionsKDFScheme`?

Reply:
It sounds a bit like it is about an actual scheme.

I am not yet to used to naming traits, but Scrypto and RET names some with prefix "Is", so I tried copying that style

Reply:
ahh ok, I was thinking if there is a pattern using `Is` prefix, it is established convention in other project as well, it makes sense